src/screens/authentication/create_new_wallet/RecoveryPhraseScreen/index.tsx

```diff
@@ -54,6 +67,37 @@ const RecoveryPhraseScreen = () => {
     <CLayout>
       <CHeader title={'Recovery Phrase'} />
       <View style={styles.container}>
+        <Row.LR pt={16} px={16}>
+          <View style={styles.selectType}>
+            <Text style={styles.algorithmLabel}>Select Algorithm</Text>
```

Comment:
Please change the label to "Choose Encryption Type"

Reply:
hello @hoailinh1210 , please have a look. Thanks.

Reply:
@gaupoit @hoailinh1210 we also need to have a small text to explain differences between 2 types

> We recommend to choose ed25519 over secp256k1 for stronger security and better performance, unless you explicitly want to use secp256k1 in order to compatible with Bitcoin, Ethereum chains

Reply:
It's valid suggestion. Thanks @codsay . Cc @hoailinh1210 @djhahe .